content/docs/reference/design-docs/train-sim-v3/overview.en.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Overview"
+linkTitle: "Overview"
+weight: 10
+---
+
+{{% pageinfo color="warning" %}}
+This work is pending implementation, and has not yet been adjusted to
+reflect potential required adjustments.
+{{% /pageinfo %}}
+
+After two years of extending a fairly simple simulation engine, it appeared that fundamental
+changes are required to meet expectations.
+
+
+## System requirements
+
+The new system is expected to:
+
+1. handle reactions to signaling
+
+1. handle rich train state (pantograph position, battery state)
```

Comment:
That's a lot of `1.`s, is it normal?

Reply:
Yes, the actual numbers are generated by Hugo, you can put any number and it would work

Reply:
I still renumbered the list, so the code could be more readable